pages/index.tsx

```diff
@@ -1,15 +1,31 @@
-import { useEffect, useState } from 'react';
-
 import Trending from '../components/trending/Trending';
 import request from '../modules/request';
 import { AppWrapper, GlobalStyle } from '../styles/index.styles';
 
-const Home = (): JSX.Element => {
-  const [data, setData] = useState([]);
+interface Community {
```

Comment:
This interface is replicated here and in the Trending component. Duplication is not good, so let's fix it! I discussed it with Ioanna, and we decided to create a new file called `types.ts` which will live inside the `src` folder and both Trending and index.tsx will import.

Reply:
Done!